packages/flame_texturepacker/lib/atlas/texture_packer_atlas.dart

```diff
@@ -19,8 +21,20 @@ class TexturePackerAtlas {
 
   /// Loads all the sprites from the atlas that resides on the [path] and
   /// returns a new [TexturePackerAtlas].
-  static Future<TexturePackerAtlas> load(String path) async {
-    final atlasData = await _load(path);
+  /// If [fromStorage] is true, the atlas will be loaded from the device's
+  /// storage instead of the assets folder.
+  static Future<TexturePackerAtlas> load(
+    String path, {
+    bool fromStorage = false,
+  }) async {
+    late final _TextureAtlasData atlasData;
```

Comment:
This doesn't have to be late, the compiler figures out that it is always set in the if-else